documentation/bolt_configuration_reference.md.erb

```diff
@@ -116,7 +116,14 @@ The `puppetfile` section of the configuration file configures how to retrieve mo
 
 ## Transport configuration options
 
-Transport configuration options can be set in both the configuration file and inventory file.
+Transport configuration options can be set in the inventory file. Options at
+the inventory's top level will apply to all targets.
+
+### The transport option
```

Comment:
Definitely work for another ticket, but I think we need to look at creating separate ref pages for each config/inventory file. This is quickly getting confusing. 😬